pkg/cassandra/config/config.go

```diff
@@ -109,68 +148,64 @@ func (c *Configuration) NewSession(logger *zap.Logger) (cassandra.Session, error
 }
 
 // NewCluster creates a new gocql cluster from the configuration
-func (c *Configuration) NewCluster(logger *zap.Logger) (*gocql.ClusterConfig, error) {
-	cluster := gocql.NewCluster(c.Servers...)
-	cluster.Keyspace = c.Keyspace
-	cluster.NumConns = c.ConnectionsPerHost
-	cluster.Timeout = c.Timeout
-	cluster.ConnectTimeout = c.ConnectTimeout
-	cluster.ReconnectInterval = c.ReconnectInterval
-	cluster.SocketKeepalive = c.SocketKeepAlive
-	if c.ProtoVersion > 0 {
-		cluster.ProtoVersion = c.ProtoVersion
+func (c *Configuration) NewCluster() (*gocql.ClusterConfig, error) {
+	cluster := gocql.NewCluster(c.Connection.Servers...)
```

Comment:
I don't recall - does this line require live connection to the backend? Seems like it shouldn't since many connection-affecting options are set afterwards. If so it can be tested 100%